src/reporting/branch.se

```diff
@@ -139,17 +148,15 @@ def isContainerForReportingToken(shadyReportingToken: address):
     legitMarket = shadyMarket
     return(legitMarket.isContainerForReportingToken(shadyReportingToken))
 
-def isParentOf(shadyChild: address):
-    branchId = shadyChild.getParentPayoutDistributionHash()
-    return(self.childBranches[branchId] == shadyChild)
```

Comment:
Yeah, it may have had something to do with the rebase, but it seems correct now.